RHEL6_7/packages/RemovedPackages/noauto_postupgrade.d/native_rpm_dep_fix.sh

```diff
@@ -22,22 +29,33 @@ is_dist_native() {
 }
 
 rm_native_broken_old_rpms() {
+    #
+    # Remove native RPMs with broken dependencies.
+    #
+    # The fce can be affected by the $RM_CUSTOM_RPM variable to remove even
```

Comment:
"fce" is a contraction of Czech "funkce".

Reply:
Thanks :-)

Reply:
fixed